### PR TITLE
Spadev2

### PR DESCRIPTION
**Pull request by jsnrdl**
New Functionnal Version of Spade, live.

### CODE REVIEW

File: ccai/nn/munit/utils.py [hunk 1]
```diff
@@ -934,9 +934,9 @@ def forward(self, x, feature_alignment=False):
 
         x = self.resnet34_8s(x)
 
-        # x = nn.functional.upsample_bilinear(input=x, size=input_spatial_dim) 0.3
-        x = nn.functional.interpolate(input=x, size=input_spatial_dim, mode="bilinear")
-
+        x = nn.functional.upsample_bilinear(input=x, size=input_spatial_dim)
+        # x = nn.functional.interpolate(input=x, size=input_spatial_dim, mode="bilinear")
```
Comment (jsnrdl) [inline ccai/nn/munit/utils.py:938]: Added Mode Bilinear for optimal mask resolution
